tools/mo/openvino/tools/mo/front/tf/conv_ext.py

```diff
@@ -16,9 +16,17 @@ class Conv2DFrontExtractor(FrontExtractorOp):
     @classmethod
     def extract(cls, node):
         attrs = tf_create_attrs(node, 2, 3)
+
+        def get_num_groups(node):
+            num_groups = node.group if 'group' in node else None
+            if num_groups is None and is_fully_defined(node.in_node(0).shape) and is_fully_defined(node.kernel_shape):
+                num_groups = node.in_node(0).shape[node.channel_dims] // node.kernel_shape[node.input_feature_channel]
```

Comment:
I think it sufficient to check that concrete dimensions are not dynamic: `node.in_node(0).shape[node.channel_dims]` and `node.kernel_shape[node.input_feature_channel]`

Reply:
agreed, and corrected these lines as we discussed